src/components/forms/editSiteForm/index.tsx

```diff
@@ -10,16 +10,29 @@ import styled from 'styled-components';
 
 interface EditSiteFormProps {
   readonly formInstance: FormInstance<EditSiteRequest>;
+  readonly onEdit: (formLat: number, formLng: number) => void;
```

Comment:
⛏️ Just a thought but how about we combine `formLat` and `formLng` into a `google.maps.LatLng` because those two are paired (i.e. the combination of `formLat` and `formLng` has a specific meaning)?